src/components/Section/SectionSelector.jsx

```diff
@@ -1,47 +1,43 @@
 import React from "react";
 import SectionRegular from "./SectionRegular";
 import SectionNoImage from "./SectionNoImage";
+import SectionCarousel from "./SectionCarousel";
 
-const SectionSelector = ({
-  variant,
-  header,
-  description,
-  image,
-  buttons,
-}) => {
+const SectionSelector = (props) => {
```

Comment:
I think it is better in this case. When you desrtuct, if you add a new prop you will have to add it to destructure statement otherwise, if you forget it, it will be undefined. I spent today bloody 20min trying to figure out why the hell cards prop is undefined until I finally got it. So decided to refactor this way for future.

Reply:
hahaha okay i see :) just wanted to know ;)